server/php/settings.php

```diff
@@ -0,0 +1,49 @@
+<?php
+return [
+    'settings' => [
+        'displayErrorDetails' => true, // set to false in production
+        'addContentLengthHeader' => false, // Allow the web server to send the content-length header
+
+        // Monolog settings
+        'logger' => [
+            'name' => 'slim-app',
+            'path' => isset($_ENV['docker']) ? 'php://stdout' : __DIR__ . '/logs/app.log',
+            'level' => \Monolog\Logger::DEBUG,
+        ],
+
+        'stripe' => [
+          // You shouldn't have to touch this
+          'apiVersion' => '2019-02-11',
+
+          // Update this path if you want to move your public folder
+          'staticDir' => __DIR__ . '/../../public/',
+
+          // Find them at https://dashboard.stripe.com/account/apikeys
+          'publishableKey' => '',
+          'secretKey' => '',
+
+          // Find it at https://dashboard.stripe.com/account/webhooks
+          'webhookSecret' => '',
+
+          // Your prefered country and currency
```

Comment:
not necessarily preferred, but rather the country of your Stripe account.

Reply:
Settings.php updated.